bindings/gumjs/gumscript-runtime-objc.js

```diff
@@ -536,7 +594,7 @@
                         }
                     }
 
-                    return cachedMethodNames;
+                    return Object.keys(objCObjectBuiltins).concat(cachedMethodNames);
```

Comment:
Hmm, it's the same way we'd auto-complete a String object; the methods will be on the prototype and presumably not part of Object.keys() if I'm not mistaken.